Makefile

```
@@ -0,0 +1,38 @@
+build: server-build agent-build
+
+server: server-build server-run
+
+server-build:
```

К таргетам Makefile'а следует добавлять аттрибут `.PHONY`: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

```suggestion
.PHONY: server-build
server-build:
```